packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -2257,6 +2259,7 @@ def _get_unanswered_questions(
     ) -> Generator[None, None, List[Dict[str, Any]]]:
         """Collect FPMM from subgraph."""
         creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"_get_unanswered_questions {creator=}")
```

Comment:
```suggestion
        self.context.logger.debug(f"_get_unanswered_questions {creator=}")
```

Reply:
I will delete this line, as it's contained in other parts of the logs.